docs/pages/admin-guides/deploy-a-cluster/deployments/gcp.mdx

```diff
@@ -117,6 +117,10 @@ storage:
 Load Balancing is required for Proxy and SSH traffic. Use `TCP Load Balancing` as
 Teleport requires custom ports for SSH and Web Traffic.
 
+GCP sets a default Load Balancer timeout of 30 seconds. You should either increase this to be longer than the Teleport Auth Service default keepalive interval of 300 seconds or decrease the Teleport `keep_alive_interval` to be lower than the GCP timeout value.
```

Comment:
This information is incomplete and not true in most cases. 

AFAIK GCP does not have a concept of timeout for Passthrough TCP/UDP LBs, not for Proxy TCP/UDP LBs. The 30 second timeout is only on Application LBs (L7/HTTP) and varies based on the target type (target pool vs NEG).

The guide instructs users to use L4 LBs so they should not have to set the timeout (and I don't even think it's possible to set the L4 Lb timeout in GCP).

Reply:
I made revisions that I believe avoid inaccurate information while still informing the reader that L7 is an available option.